WooCommerce/Classes/ViewRelated/ReusableViews/CircleSpinnerView.swift

```diff
@@ -0,0 +1,128 @@
+import UIKit
+
+/// An animatable circular spinner.
+///
+@IBDesignable
```

Comment:
I think that we need to remove `@IBDesignable` here since we will allow configuring this View from Xib or Storyboard, it is something that we should avoid since we try to configure everything possible from code (and maybe it can create confusion, making the view configuration "uneven").